StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -188,5 +193,15 @@ public extension PaymentSheet {
             // TODO
             self.isCVCRecollectionEnabledCallback = { return false }
         }
+
+        @discardableResult
+        func validate() -> Error? {
+            let errorMessage: String
+            if case .payment(let amount, _, _, _) = mode, amount > 0 {
```

Comment:
Based on the error message, shouldn't this be `if amount <=0` ?

Reply:
>_> yes